frontend/test/__support__/ui.tsx

```diff
@@ -207,11 +208,11 @@ function SdkWrapper({
   return (
     <Provider store={store}>
       <EmotionCacheProvider>
-        <ThemeProvider>
+        <SdkThemeProvider>
```

Comment:
updated the SDK wrapper to use the `SdkThemeProvider` instead to be more realistic